modules/util/portal-tools-db-schema-importer/src/main/java/com/liferay/portal/tools/db/schema/importer/jdbc/util/DataSourceFactoryUtil.java

```diff
@@ -0,0 +1,127 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.tools.db.schema.importer.jdbc.util;
+
+import com.liferay.petra.reflect.ReflectionUtil;
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.util.ArrayUtil;
+import com.liferay.portal.kernel.util.JavaDetector;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+
+import java.lang.reflect.Field;
+
+import java.util.List;
+
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLEngine;
+
+import javax.sql.DataSource;
+
+/**
+ * @author Mariano Álvaro Sáiz
+ */
+public class DataSourceFactoryUtil {
```

Comment:
I don't think we need this class since most of the logic can be removed, we will use this tool internally so we won't use the IBM JVM, it is internal and also we are using MySQL Connector/J 8.0.20 and above and that's the recommendation of LPS-120753.

So we should have just one method to initHikari in the main class.

@marianoalvarosaiz let me know if I miss something.

Reply:
OK, if we discard the IBM JVM then is fine for me.